Bottom: d60ef3303ef67bd32a55b25bf032f81023ccef52
Top:    635d3ea38e37045e9c6e6d10e00d26a7e26ae2eb
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-10-21 11:59:22 +0100

Refresh of set-changed-flag-on-pcb-when-l

---

diff --git a/src/action.c b/src/action.c
index 6e5c25c..bc1cc40 100644
--- a/src/action.c
+++ b/src/action.c
@@ -1151,7 +1151,6 @@ NotifyMode (void)
 	    DrawElement (element);
 	    Draw ();
 	    SetChangedFlag (true);
-	    hid_actionl ("Report", "Object", NULL);
 	  }
 	else if (type != NO_TYPE)
 	  {
@@ -1166,7 +1165,6 @@ NotifyMode (void)
 		Draw ();
 	      }
 	    SetChangedFlag (true);
-	    hid_actionl ("Report", "Object", NULL);
 	  }
 	break;
       }
